test/compile-error-test.bats.template

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bats
+# Compile error test
+# 
+# The test case here feeds a malformed .ddl into ddlog's print command and compares whether it produces an expected error.
+
+source bats-template.bash  # for $TESTDIR, $it, etc.
+
+# some preconditions
+setup() {
+    [ -e "$TESTDIR" ]
+    expectedError="$TESTDIR"/compile-error.expected
+    [ -e "$expectedError" ]
+    actualError=${expectedError%.expected}.actual
+}
+
+# check if input produces a compile error
+@test "$it compiles input" {
+    ! ddlog compile "$TESTDIR"/input.ddl >/dev/null 2>"$actualError"
```

Comment:
Yes, that's the current design. I'll change this to `checker`.

Sorry, miss the print...